servers/services/src/main/java/org/projectnessie/services/impl/NamespaceApiImpl.java

```diff
@@ -159,31 +171,32 @@ public Namespace getNamespace(NamespaceParams params)
    * 'a.b.c.table' where the 'a.b.c' part * represents the namespace.
    *
    * @param namespace The namespace to fetch
-   * @param branch The ref to use
+   * @param refWithHash The ref to use
    * @return A {@link Namespace} instance
    * @throws ReferenceNotFoundException If the ref could not be found
    * @throws NessieNamespaceNotFoundException If the namespace could not be found
    */
-  private Namespace getNamespace(Namespace namespace, BranchName branch)
+  private Namespace getNamespace(Namespace namespace, WithHash<NamedRef> refWithHash)
```

Comment:
The function only uses the `Hash` - maybe better to refactor to `Hash hash` (the name's irrelevant)